README.md

```diff
@@ -1,14 +1,8 @@
 # Jitterbug
-A Laravel / MySQL database management application to support large-scale description, digitization, preservation and access of archival audiovisual recordings in the Wilson Special Collections Library of UNC-Chapel Hill. Funded by an Andrew W. Mellon Foundation grant, "Extending the Reach of Southern Audiovisual Sources."
+A Laravel / MySQL database management application to support large-scale description, digitization, preservation and 
+access of archival audiovisual recordings in the Wilson Special Collections Library of UNC-Chapel Hill. Funded by an 
```

Comment:
Some weird whitespace and linebreaks here and in various places in this PR.

Reply:
i returned all those long lines back into single lines for easier reading on github =)